src/include/grpc_transcoding/path_matcher.h

```diff
@@ -151,8 +151,7 @@ class PathMatcherBuilder {
   std::unordered_set<std::string> custom_verbs_;
   typedef typename PathMatcher<Method>::MethodData MethodData;
   std::vector<std::unique_ptr<MethodData>> methods_;
-  bool always_decode_;
-  bool fully_decode_reserved_expansion_;
+  UrlUnescapeSpec unescape_spec_;
```

Comment:
init it with a specific default

Reply:
I've specified default in `PathMatcherBuilder` constructor, can move here for clarity.